itests/org.jboss.tools.cdk.ui.bot.test/src/org/jboss/tools/cdk/ui/bot/test/server/adapter/CDKServerAdapterAbstractTest.java

```diff
@@ -233,12 +237,12 @@ protected void startServerAdapter(Runnable cond, boolean rethrow) {
 	protected void startServerAdapter(Server server, Runnable cond, boolean rethrow) {
 		log.info("Starting server adapter"); 
 		// Workaround for CDK-216
-		new WaitUntil(new NeverFulfilledCondition(), TimePeriod.getCustom(120), false);
+		// new WaitUntil(new NeverFulfilledCondition(), TimePeriod.getCustom(120), false);
```

Comment:
@odockal can it be deleted? CDK-216 is closed.

Reply:
Probably yes!